apps/hash-graph/lib/graph/src/knowledge/entity/query.rs

```diff
@@ -58,22 +58,22 @@ pub enum EntityQueryPath<'p> {
     /// [`EntityRecordId`]: crate::identifier::knowledge::EntityRecordId
     /// [`Entity`]: crate::knowledge::Entity
     EditionId,
-    /// The decision time axis of the [`EntityVersion`] belonging to the [`Entity`].
+    /// The decision time axis of the [`EntityTemporalMetadata`] belonging to the [`Entity`].
     ///
-    /// To query for an [`EntityVersion`] the time projection is specified on the
+    /// To query for an [`EntityTemporalMetadata`] the time projection is specified on the
     /// [`StructuralQuery`].
     ///
     /// [`StructuralQuery`]: crate::shared::subgraph::query::StructuralQuery
-    /// [`EntityVersion`]: crate::identifier::knowledge::EntityVersion
+    /// [`EntityTemporalMetadata`]: crate::identifier::knowledge::EntityTemporalMetadata
     /// [`Entity`]: crate::knowledge::Entity
     DecisionTime,
-    /// The transaction time axis of the [`EntityVersion`] belonging to the [`Entity`].
+    /// The transaction time axis of the [`EntityTemporalMetadata`] belonging to the [`Entity`].
     ///
-    /// To query for an [`EntityVersion`] the time projection is specified on the
+    /// To query for an [`EntityTemporalMetadata`] the time projection is specified on the
     /// [`StructuralQuery`].
```

Comment:
Same as above

Reply:
Clarified in [Clarify documentation on query parameters](https://github.com/hashintel/hash/pull/1854/commits/a6e340c956289e29356134dbc95e48749cbe82dd)